_layouts/post.html

```diff
@@ -26,5 +27,7 @@ <h1>{{ page.title }}</h1>
 <div class="share-wrapper">
   {% include social-media-share-links.html %}
 </div>
+<hr>
```

Comment:
It looks like this particular hr doesn't span the whole view width, just the content container.